src/Cell.tsx

```diff
@@ -133,21 +134,30 @@ function Cell<R>({
     style.transform = `translateX(${scrollLeft}px)`;
   }
 
+  if (!children) {
+    const { formatter } = column;
+
+    if (isValidElementType(formatter)) {
```

Comment:
IMO we don't need to bother with this check.
```tsx
if (!children) {
  children = createElement<typeof formatterProps>(column.formatter, {
    value: rowData[column.key],
    column,
    rowIdx,
    row: rowData,
    isRowSelected,
    onRowSelectionChange,
    isSummaryRow
  });
}
```

We should also remove support for the `children` prop on `Cell`, maybe in the future.

Reply:
Do you think it is better to pass down `defaultFormatter` instead setting the formatter on the calculated column? We have two options 
- Make formatter a required prop in `CalculatedColumn`
- or use `const formatter = column.formatter || props.defaultFormatter`

Reply:
I think I'd rather make the `formatter` be a required type on `CalculatedColumn`, instead of prop drilling.